static/js/sections.js

```diff
@@ -1,28 +1,107 @@
 /** GROUPS **/
 
+let agent_interval_time = 30000;
+let agent_interval = null;
+
 $(document).ready(function () {
-    $('#netTbl tbody').on( 'click', 'tr', function () {
-        $(this).toggleClass('selected');
-    });
     $('#netTbl').DataTable({
-        columnDefs: [{
-            orderable: false,
-            className: 'select-checkbox',
-            targets: 0
-        }],
+        ajax: {
+            url: '/plugin/chain/rest',
```

Comment:
i tried a few things i would have anticipated working:

1. Watching the 'manage groups' tab, i would have expected the agent check-ins to increment without me refreshing the page
2. Starting a new operation, I would have anticipated the operation would appear in the drop-down without requiring a manual refresh

am i missing something?

Reply:
Both of those use cases work on the branch...